lib/hergetto/helpers/service_start_helper.ex

```diff
@@ -1,10 +1,11 @@
 defmodule Hergetto.Helpers.ServiceStartHelper do
-  def start(supervisor, service) do
+  def start(supervisor, service, room) do
     id = UUID.uuid4()
-    case DynamicSupervisor.start_child(supervisor, {service, id}) do
+    case DynamicSupervisor.start_child(supervisor, {service, {id, room}}) do
       {:ok, _pid} ->
         {:ok, id}
-      _ ->
+      d ->
+        IO.inspect(d)
```

Comment:
please remove or add something along the lines of `Logger.error("failed to start service: #{id}")`

Reply:
Will fix this (this was still from debugging xD)